networking/multiple_networks/understanding-multiple-networks.adoc

```diff
@@ -6,57 +6,67 @@ include::_attributes/common-attributes.adoc[]
+
+By creating multiple networks within a cluster, you gain greater flexibility in network design, as well as enhanced security, scalability, and workload isolation. Multiple networks allow you to configure network topologies tailored to your specific requirements, such as multi-tenancy, traffic isolation, and fine-grained connectivity control between workloads. Multiple networks can be implemented at the namespace level, and as both primary and secondary networks. For primary networks, user-defined networks (UDNs) and network attachment definitions (NADs) are available. For secondary networks, cluster administrators can use NADs or one of several CNI plugins that are available; support for secondary networks with UDN will be added in a future version of {product-title}. Additional CNIs available can be viewed under the "Additional networks in {product-title}" section.
+
+[id="primary-networks_{context}"]
+== Primary networks
+
+In {product-title}, the term _primary network_ can refer to one of two things:
+
+// The info about CUDN will be added in a separate PR now that SDN-4930 has been merged.
+* Cluster-wide primary network. This network is global to the entire cluster. When configured, all namespaces use it unless overridden. This network is managed by OVN-Kubernetes, and is configured during cluster installation. {product-title} administrators can configure UDN to serve as the entire cluster's primary network. This option is not available for NADs. For more information about configuring UDN as your cluster-wide primary network, see. . . 
+
+//xref will lead to CUDN content in a separate PR.
+
+* Namespace-scoped primary networks. When configuring namespace-scoped primary networks, {product-title} administrators can choose between NADs or UDNs; {product-title} users are also provided the ability to configure multiple networks for a namespace with UDNs. A combination of NADs or UDNs can also be implemented within a deployment if warranted. For example, an administrator might choose to use a NAD for one namespace, and a UDN for another. 
+
+The following sections provide more information about UDNs and NADs.
+
+[id="user-defined-networks_{context}"]
+=== User-defined networks
+
+User-defined networks offer enhanced customization and isolation by leveraging the `ClusterUserDefinedNetwork` custom resource (CR) or the `UserDefinedNetwork` CR depending on your needs. Unlike NADs, which are only namespaced scope, UDNs offer administrators the ability to create and define additional networks at the cluster level. UDNs also offer both administrators and users the ability to define additional networks at the namespace level.
+
+User-defined networks can be customized with Layer 2 or Layer 3 topology types, enabling a wide range of network architectures and topologies. 
+
+[NOTE]
```

Comment:
I'd cut this promise of future support from the doc. I get where it is heading though and the impetus to add it.